sunpy/map/sources/suit.py

```diff
@@ -0,0 +1,121 @@
+"""
+SUIT Map sub class definitions
+
+"""
+from sunpy.map.sources.source_type import source_stretch
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import CartesianRepresentation, HeliocentricMeanEcliptic
+from sunpy.map.mapbase import GenericMap
+from matplotlib import colors
+from astropy.visualization import ImageNormalize
+from astropy.visualization import AsinhStretch
+
+__all__ = ['SUITMap']
+__author__ = ['Rahul Gopalakrishnan']
+__email__ = ['rahulg.astro@gmail.com']
+
+class SUITMap(GenericMap):
+    """
+    The Solar Ultraviolet Imaging Telescope (SUIT) is one of the remote sensing payloads on board the Aditya-L1 mission of the Indian Space Research Organization (ISRO)
+    that was launched on September 02, 2023. SUIT is designed to provide near-simultaneous full-disk and region-of-interest images of the Sun at various heights,
+    slicing through the photosphere and chromosphere, employing an array of 11 scientifically calibrated filters (3 broad-band & 8 narrow-band) strategically
+    positioned within the wavelength range of 200 to 400 nanometers. Located at the first Lagrange point, SUIT observes the Sun 24x7, without any interruption.
```

Comment:
The preferred format for a docstring is one short (<80 character) description, a blank line and then more details. So something like
```
The Solar Ultraviolet Imaging Telescope (SUIT)

<more stuff here>
```